source/src/cip/cipconnectionmanager.h

```diff
@@ -159,8 +159,8 @@ typedef enum {
 #define SEQ_LEQ16(a, b) ( (short)( (a) - (b) ) <= 0 )
 #define SEQ_GEQ16(a, b) ( (short)( (a) - (b) ) >= 0 )
 
-/** @brief Connection Manager class code */
-static const int g_kCipConnectionManagerClassCode = 0x06;
+/** @brief Connection Manager class id */
+static const EipUint16 kCipConnectionManagerClassId = 0x06u;
```

Comment:
use CipUint

Reply:
Here the constant keeps the "ClassCode" prefix but loses the "g_" prefix. Needs also comment revert.